dashboard/test/ui/features/teacher_tools/levelbuilder/lesson_edit_page.feature

```diff
@@ -71,7 +71,7 @@ Feature: Using the Lesson Edit Page
     And I press ".fa-search" using jQuery
     # We will know the search has completed after the following step, because we
     # confirmed earlier that there were no Artist levels in the initial view.
-    And I wait until element ".uitest-level-dialog-content td" contains text "Artist"
+    And I wait until element ".uitest-level-dialog-content td" contains text "Standalone_Artist_1"
```

Comment:
Can you update the comments in this scenario? We have a comment above and here referring to "Artist", we should update to explain why we think search succeeded if we see "Standalone_Artist_1".